patches/chrome-browser-ui-webui-settings-md_settings_ui.cc.patch

```diff
@@ -1,13 +1,14 @@
 diff --git a/chrome/browser/ui/webui/settings/md_settings_ui.cc b/chrome/browser/ui/webui/settings/md_settings_ui.cc
-index fa6dd8d8b36b98e60a288a1f532e0c066038f0c4..9379a37df65a369ab445dcabbcc0692cb12ee83f 100644
+index fa6dd8d8b36b98e60a288a1f532e0c066038f0c4..3f9d4b2157505ccb7dc4b86fc8fb15687402e8ab 100644
 --- a/chrome/browser/ui/webui/settings/md_settings_ui.cc
 +++ b/chrome/browser/ui/webui/settings/md_settings_ui.cc
-@@ -137,6 +137,8 @@ void MdSettingsUI::RegisterProfilePrefs(
+@@ -137,6 +137,9 @@ void MdSettingsUI::RegisterProfilePrefs(
    registry->RegisterBooleanPref(prefs::kImportDialogHistory, true);
    registry->RegisterBooleanPref(prefs::kImportDialogSavedPasswords, true);
    registry->RegisterBooleanPref(prefs::kImportDialogSearchEngine, true);
```

Comment:
this entire patch should be removed and put the pref registrations in brave_profile_prefs.cc